playwright-e2e/dss/component/select.ts

```diff
@@ -40,8 +46,9 @@ export default class Select extends WidgetBase {
    * @param opts
    * @returns {Promise<void>}
    */
-  async selectOption(value: string, opts: { exactMatch?: boolean } = {}): Promise<void> {
-    const { exactMatch = true } = opts;
+  async selectOption(value: string, opts: { exactMatch?: boolean, nth?: number } = {}): Promise<void> {
+    const { exactMatch = true, nth = 0 } = opts;
+    await expect(this.toLocator()).toBeEnabled();
```

Comment:
Adding new optional param `nth?: number` and locator `toBeEnabled()` check to prevent flakiness.